contracts/.env.example

```diff
@@ -1,5 +1,8 @@
-# The RPC API URL (e.g. https://rpc-mumbai.maticvigil.com https://polygon-mumbai.g.alchemy.com/v2/your-api-key)
-API_URL=
+# The RPC API URL (e.g. https://polygon-mainnet.g.alchemy.com/v2/your-api-key)
+ETH_SEPOLIA_API_URL=
```

Comment:
The only value I found unnecessary was the sepolia etherscan key since it would be the same as the etherscan key value.

Polyscan's key differs from etherscan so I don't think we should remove that.

And I don't think we need to remove all RPC URLs and then filter their value to deduce what network it's for. I might be missing the point here so please correct me if I'm wrong. Also, the explorer URL is different for all of them.

It's also worth noting that hardhat requires all of these values in its config to function even if the user tries to run them locally.

Reply:
It's more from env var practices. When do we do this in other apps? I.e., keep creating new variable names for the same variable but based on the value? We don't go:

DYNAMO_ENDPOINT_LOCAL
DYNAMO_ENDPOINT_DEV
DYNAMO_ENDPOINT_PRD

We just go DYNAMO_ENDPOINT and it's value gets set right? That's my point here